ipn/backend.go

```diff
@@ -178,21 +178,11 @@ type StateKey string
 type Options struct {
```

Comment:
There's some `StateKey` comments right above this that need updating (as far as removing `_daemon` references, etc.)